web/tests/settings_org.test.js

```diff
@@ -630,6 +630,13 @@ test("set_up", ({override, override_rewire}) => {
         $.create("<stub-create-web-public-stream-policy-parent>"),
     );
 
+    override_rewire(settings_components, "get_input_element_value", (elem) => {
+        if ($(elem).data() === "number") {
+            return Number.parseInt($(elem).val(), 10);
+        }
+        return $(elem).val();
+    });
```

Comment:
Mocked this function as the test gave `ReferenceError: HTMLInputElement is not defined` error on one of the assert calls in `get_input_element_value`, not sure how to fix this in other way.